prime-dotnet-webapi/HttpClients/LDAP Api/LdapClient.cs

```diff
@@ -0,0 +1,92 @@
+using System;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json.Linq;
+
+namespace Prime.HttpClients
+{
+    public class LdapClient : BaseClient, ILdapClient
+    {
+        private readonly HttpClient _client;
+        private readonly ILogger _logger;
+
+        public LdapClient(
+            HttpClient client,
+            ILogger<LdapClient> logger)
+            : base(PropertySerialization.CamelCase)
+        {
+            // Auth header and api-key are injected in Startup.cs
+            _client = client;
+            _logger = logger;
+        }
+
+        public async Task<JObject> GetUserAsync(string username, string password)
+        {
+            JObject messageObject = new JObject
```

Comment:
Does this have to be a JObject? I think we should avoid using JObject when its just as convienient to do:
```C#
var messageObject = new 
{
    UserName = username,
    Password = password
}
```